examples/iam-read-only-policy/main.tf

```diff
@@ -0,0 +1,39 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+locals {
+  allowed_services_compute     = ["ec2", "ecr", "eks", "ecs", "lambda", "autoscaling", "elasticloadbalancing"]
+  allowed_services_networking  = ["vpc", "route53", "route53domains", "route53resolver", "servicediscovery"]
+  allowed_services_storage     = ["s3", "backup", "dynamo", "dms", "elasticfilesystem"]
+  allowed_services_databases   = ["rds", "dynamo", "elasticache"]
+  allowed_services_management  = ["cloudwatch", "events", "logs", "servicequotas", "ssm"]
+  allowed_services_analytics   = ["es", "firehose", "kinesis", "kinesisanalytics", "redshift"]
+  allowed_services_application = ["ses", "sns", "sqs", "xray", "applicationinsights", "application-autoscaling"]
+  allowed_services_security    = ["iam", "acm", "kms", "secretsmanager"]
+
+  allowed_services = concat(
```

Comment:
`distinct(concat(...))` to be safer :)

Reply:
added distinct to the module itself to sanitize input

Reply:
thus here it will be unnecessary